src/scripts/main.js

```diff
@@ -1,3 +1,68 @@
 'use strict';
 
-// write code here
+document.addEventListener('DOMContentLoaded', () => {
+  const table = document.querySelector('.field');
+  const appendRowButton = document.querySelector('.append-row');
+  const removeRowButton = document.querySelector('.remove-row');
+  const appendColumnButton = document.querySelector('.append-column');
+  const removeColumnButton = document.querySelector('.remove-column');
+
+  const MIN = 2;
+  const MAX = 10;
+
+  function updateButtons() {
+    const rowCount = table.rows.length;
+    const columnCount = table.rows[0].cells.length;
```

Comment:
Ensure that the table has at least one row before accessing `table.rows[0]` to avoid potential errors if the table is empty. Consider adding a check or initializing the table with a minimum number of rows.